src/components/dialogs/parameters/voltageinit/equipment-selection-parameters.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { elementType } from '@gridsuite/commons-ui';
+import { Box, DialogContent, Grid, Typography } from '@mui/material';
+import { EQUIPMENT_TYPES } from 'components/utils/equipment-types';
+import {
+    FIXED_GENERATORS,
+    VARIABLE_SHUNT_COMPENSATORS,
+    VARIABLE_TRANSFORMERS,
+} from 'components/utils/field-constants';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useStyles } from '../parameters';
+import DirectoryItemsInput from '../../../utils/rhf-inputs/directory-items-input';
+
+const EquipmentSelectionParameters = () => {
+    const classes = useStyles();
+
+    return (
+        <DialogContent>
+            <Grid item container>
```

Comment:
```suggestion
            <Grid item container rowSpacing={1}>
                <Grid item container>
```
I think it would be prettier and closer to what the PO has suggested

Reply:
Yes it was too compact. I added "spacing={1} padding={1}"